src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.NonAndroid.cs

```diff
@@ -74,52 +100,68 @@ private static TimeZoneInfoResult TryGetTimeZoneFromLocalMachineCore(string id,
         /// <remarks>
         /// Lines that start with # are comments and are skipped.
         /// </remarks>
-        private static List<string> GetTimeZoneIds()
+        private static IEnumerable<string> GetTimeZoneIds()
+        {
+#if TARGET_WASI || TARGET_BROWSER
+                byte[]? rawData = null;
+                if (UseEmbeddedTzDatabase)
+                {
+                    if(!TryLoadEmbeddedTzFile(TimeZoneFileName, ref rawData))
```

Comment:
Use `out` here, and the `= null` initialization for `rawData` can be dropped too.